bundle/manifests/external-dns-operator_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -71,19 +71,6 @@ rules:
   - get
   - list
   - watch
-- apiGroups:
```

Comment:
Was this reflected after running make bundle-image-build ?

Reply:
this was reflected from running `make olm-manifests` really, after removing kubebuilder rbac annotations on `operator.go`